documentation/api/query/v4/packages.markdown

```diff
@@ -0,0 +1,86 @@
+---
+title: "PuppetDB 4.4: Packages endpoint"
+layout: default
+canonical: "/puppetdb/latest/api/query/v4/packages.html"
+---
+
+[curl]: ../curl.html#using-curl-from-localhost-non-sslhttp
+[paging]: ./paging.html
+[query]: ./query.html
+[subqueries]: ./ast.html#subquery-operators
+[facts-format]: ../../wire_format/
+[ast]: ./ast.html
+[pql]: ./pql.html
+[factsets]: ./factsets.html
+[reports]: ./reports.html
+[catalogs]: ./catalogs.html
+[nodes]: ./nodes.html
+[facts]: ./facts.html
+[fact_contents]: ./fact_contents.html
+[events]: ./events.html
+[edges]: ./edges.html
+[resources]: ./resources.html
+[inventory]: ./inventory.html
+
+Package metadata collection, storage, and query is primarily a Puppet Enterprise
+feature. This data is queryable from PuppetDB if it is submitted via the
+`package_inventory` key in the payload of the `store facts` command, as
+described in the [facts format documentation.][facts-format]
```

Comment:
Are we updating the wireformat for this? That's what the wording here suggests to me